features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/webpack.config.js

```diff
@@ -118,5 +118,14 @@ module.exports = function (env) {
         const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
         config.plugins.push(new BundleAnalyzerPlugin());
     }
+    if (env && env.unused) {
+        const { UnusedFilesWebpackPlugin } = require('unused-files-webpack-plugin');
+
+        config.plugins.push(new UnusedFilesWebpackPlugin({
+            failOnUnused: process.env.NODE_ENV !== 'development',
+            patterns: ['source/src/**/*.jsx', 'source/src/**/*.js'],
```

Comment:
This will take, test files and worker files as well in publisher ?

Reply:
Haven't applied this to publisher yet. Will keep in mind when applying to Publisher